politeiawww/cmd/pictl/time.go

```diff
@@ -21,3 +21,14 @@ func timestampFromUnix(unixTime int64) string {
 	t := time.Unix(unixTime, 0)
 	return t.Format(timeFormat)
 }
+
+// unixFromTimestamp converts a human readable timestamp string formatted
+// according to the timeFormat global variable into a unix timestamp.
+func unixFromTimestamp(timestamp string) (int64, error) {
```

Comment:
We probably want to base this one the user's local timezone so as to not cause confusion.  Here's an example from some code I wrote for a side project.

```
const (
	// timeFormat contains the reference time format that is used
	// throughout this CLI tool. This format is how a timestamp should
	// be formatted when provided as a command argument and is how
	// timestamps are printed when we want to print the human readable
	// version.
	//
	// Mon Jan 2 15:04:05 -0700 MST 2006
	timeFormat = "01/02/2006 3:04pm"

	// locationName is the name of the time zone location that is used
	// in the human readable timestamps.
	locationName = "Local"
)

var (
	// location is the time zone location that is used in the human
	// readable timestamps. location is set in init().
	location *time.Location
)

// timestampFromUnix converts a unix timestamp into a human readable timestamp
// string formatted according to the timeFormat global variable.
func timestampFromUnix(unixTime int64) string {
	t := time.Unix(unixTime, 0)
	return t.Format(timeFormat)
}

// unixFromTimestamp converts a timestamp string that is in the format
// specified by the timeFormat global variable into a unix timestamp.
func unixFromTimestamp(timestamp string) (int64, error) {
	t, err := time.ParseInLocation(timeFormat, timestamp, location)
	if err != nil {
		return 0, err
	}
	return t.Unix(), nil
}

func init() {
	var err error
	location, err = time.LoadLocation(locationName)
	if err != nil {
		panic(err)
	}
}
```